packages/botonic-react/src/components/message.jsx

```diff
@@ -98,10 +104,16 @@ export const Message = props => {
     }
 
     let BotMessageImage = Logo
-    if ('brandImage' in webchatState.theme)
-      BotMessageImage = webchatState.theme.brandImage
-    if ('botMessageImage' in webchatState.theme)
-      BotMessageImage = webchatState.theme.botMessageImage
+    // In this condition we look for 'brand.image' or 'brandImage' to exist and to have value (which can be null)
+    if (getProperty(theme, 'brand.image') || getProperty(theme, 'brandImage')) {
```

Comment:
I have refactored this part @guinii. I have modified a little bit the method `getThemeProperty` as well in order to fix a little issue with attributes explicitly set to null. Typical case: you don't want to display the headerImage because you have set it to `null`.